### ISSUE TITLE
Button disapears (almost)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#351]
```
What's the problem?

It was working fine until this few days.  The buttons becomes very thin without 
labels visable.


What browser are you using?


Firefox 4


What version of ScribeFire are you running?

1.6 latest on mozilla


Where do you store blog accounts and passwords?  When I remove and reinstall 
scribefire, I can still access the blogs.  I deleted the sqlite for example.
```

-----
Original issue reported on code.google.com by `client2...@gmail.com` on 23 May 2011 at 6:18

**Comment by GoogleCodeExporter** [comment]
```
The error console reports invalid/privilege URI access, uncatched exception.  
BTW my profile directory is in NTFS partition shared with windows.  It was 
working fine.
```

-----
Original comment by `client2...@gmail.com` on 23 May 2011 at 6:21

**Comment by GoogleCodeExporter** [comment]
```
I reinstalled the 1.5 version for firefox and works great as always.  The 1.6 
version had many warnings when fired up.  The exception was generated trying to 
read the en-US locale file.  So there was no labels on the buttons and they 
almost disappear.
```

-----
Original comment by `client2...@gmail.com` on 23 May 2011 at 9:01

**Comment by GoogleCodeExporter** [comment]
```
Can you take a screenshot of the errors from 1.6? I know the general reason why 
they're happening, but seeing the exact errors would help a lot.
```

-----
Original comment by `cfinke` on 24 May 2011 at 4:06
* Changed state: **Pending**

**Comment by GoogleCodeExporter** [comment]
```
Oh I understand the reason at last.
It was a big deal at Mozilla that they change something about privileges system 
after two years of planning.
Same origin policy of xmlhttprequest extends to local chrome:// files.
Scribefire just happen to use that to read locale files.
The request simply fail and generates an uncaught exception.
The user end up with an UI without text, so the empty buttons are very thin.

I'm using FF5 so there should be no simple way to allow multiple origins.
In FF5 I can't even go back to older Scribefires versions using the old locale 
handling, unless I edit the max version in the install file.

I'm using Linux but it's an issue with FF.  So it that works in Windows & Mac I 
don't understand.
```

-----
Original comment by `CCAScien...@gmail.com` on 10 Jul 2011 at 3:51

**Comment by GoogleCodeExporter** [comment]
```
Yes, there's no UI text at all for the latest scribefire's, as long as they use 
xmlhttprequest.  Because they can't read the locale file.
```

-----
Original comment by `client2...@gmail.com` on 11 Jul 2011 at 4:51

**Comment by GoogleCodeExporter** [comment]
```
This has been fixed in version 1.8.
```

-----
Original comment by `cfinke` on 19 Jul 2011 at 2:18
* Changed state: **Fixed**
